src/devices/yale.ts

```diff
@@ -424,6 +424,13 @@ const definitions: DefinitionWithExtend[] = [
         description: 'Real living lock / Intelligent biometric digital lock',
         extend: [lockExtend({battery: {dontDividePercentage: true}})],
     },
+    {
+        zigbeeModel: ['06e0152204'],
+        model: 'YMI70A',
+        vendor: 'Yale',
+        description: 'Biometric digital lock',
+        extend: [battery(), lock({pinCodeCount: 1})],
```

Comment:
Is `pinCodeCount` `1` correct?

Reply:
Sorry! The device can handle up to 100 pin codes.
My bad. I'll fix it.